opencensus/trace/trace_options.py

```diff
@@ -18,40 +18,43 @@
 _ENABLED_BITMASK = 1 << 0
 
 # Default trace options
-DEFAULT = '1'
+DEFAULT = 0x0
 
 
 class TraceOptions(object):
     """A class that represents global trace options.
 
-    :type trace_options_byte: str
+    :type trace_options_byte: int
     :param trace_options_byte: 1 byte bitmap for trace options.
     """
 
-    def __init__(self, trace_options_byte=None):
-        if trace_options_byte is None:
+    def __init__(self, trace_options_byte=DEFAULT):
+        if not isinstance(trace_options_byte, int):
+            logging.warning("trace_options_byte must be an int, this will be "
+                            "an error in future versions")
```

Comment:
This is technically an API change, so I figure the right approach is to log a warning here and make this an error in the next minor version release. We would benefit from some convention for deprecation, compare to java's `@deprecated` annotation.